pkgs/by-name/do/dooit/package.nix

```diff
@@ -34,6 +33,21 @@ python3.pkgs.buildPythonApplication rec {
     python-dateutil
     pyyaml
     textual
```

Comment:
```suggestion
```

Reply:
Looks like the other reference got left in there, so it's picking up the new version, as well.

Reply:
oh lol hahaha